### PR TITLE
draggable keeper, positions remembered per user+site

### PR DESCRIPTION
**Pull request by 2is10**

### CODE REVIEW

**Comment by 2is10** [comment]
This is done except for making the hover tips show up below the keeper when it's near the top of the window. I'll reopen when that's addressed.

File: packrat/styles/metro/slider2.css [hunk 1]
```diff
@@ -479,3 +495,13 @@ a.kifi-slider2-keeper {
...
   padding-top: 6px;
 }
+
+.kifi-slider2-drag-glass {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 999999999993;
```
Comment (andrewconner) [inline packrat/styles/metro/slider2.css:505]: Nice, to be bigger than all those 999999999992's :)

Comment (2is10) [inline packrat/styles/metro/slider2.css:505]: You'd be surprised. Hab.la uses ten 9's. I'm using eleven plus a final digit that varies according to our needs.

**Comment by 2is10** [comment]
Danny wants to launch without fixing the hovertips.

**Comment by andrewconner** [comment]
Amazing.

**Pull request merged**